core/lib/types/src/commitment.rs

```diff
@@ -209,28 +253,54 @@ struct L1BatchAuxiliaryOutput {
     initial_writes: Vec<InitialStorageWrite>,
     #[allow(dead_code)]
     repeated_writes: Vec<RepeatedStorageWrite>,
+
     l2_l1_logs_compressed: Vec<u8>,
     l2_l1_logs_linear_hash: H256,
     l2_l1_logs_merkle_root: H256,
+
+    // Once cut over to boojum, these fields are no longer required as their values
```

Comment:
Would it make sense to create a task to get rid of these fields then? Otherwise we're risking having them here forever.

Reply:
i think so, @perekopskiy @RomanBrodetski what do you think and should i make this a platform or protocol task?

Reply:
It will be relevant to l1_batches table split/clean up, feel free to create platform task